README.md

```diff
@@ -106,6 +106,30 @@ Currently, we support the following types of assets:
 - [ERC-721](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/) (Non-fungible tokens)
 - [ERC-1155](https://ethereum.org/en/developers/docs/standards/tokens/erc-1155/) (Multi-tokens)
 
+### Input encodings for deposits
+
+As explained above, when a portal teleports assets to the execution layer, it adds an input to the DApp's input box, which will then need to be interpreted and validated by the off-chain machine. In order to do that, the machine will need to understand and decode the input's payload.
+
+The input payloads for deposits are always specified as packed ABI-encoded parameters, as detailed below:
+
+| Asset | Payload |
+| :- | :- |
+| Ether | `address sender, uint256 value, bytes execLayerData` |
+| ERC-20 | `bool success, address token, address sender, uint256 amount, bytes execLayerData` |
+| ERC-721 | `address token, address sender, uint256 tokenId, bytes data` `*` |
+| ERC-1155 (single) | `address token, address sender, uint256 tokenId, uint256 value, bytes data` `*`|
+| ERC-1155 (batch) | `address token, address sender, bytes data` `**` |
+
+**`*`** The ERC-721 and ERC-1155 (single) `data` fields correspond to the ABI-encoding of: `bytes baseLayerData, bytes execLayerData`
```

Comment:
I think it's healthy to highlight here that for these `data` fields, are using the **standard** ABI encoding, and not the packed one.